.github/workflows/Changelog.yml

```diff
@@ -0,0 +1,29 @@
+name: Changelog
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Conventional Changelog Action
+        id: changelog
+        uses: TriPSs/conventional-changelog-action@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          output-file: "false"
+
+      - name: Create Release
+        uses: actions/create-release@v1
+        if: ${{ steps.changelog.outputs.skipped == 'false' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.changelog.outputs.tag }}
+          release_name: ${{ steps.changelog.outputs.tag }}
+          body: ${{ steps.changelog.outputs.clean_changelog }}
```

Comment:
where these values passed on?

Reply:
It is passing the generated changelog and the tag information to the TriPSs/conventional-changelog-action@v3 action, which might use this information to create a GitHub release, attach the changelog to a release, or perform other tasks related to versioning and release management. The specific functionality depends on how the TriPSs/conventional-changelog-action@v3 action is implemented.